src/components/Menu/bubble.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <div>
-        <Menu v-bind="propsToPass">
+        <Menu v-bind="propsToPass" openMenu="openMenu" @closeMenu="closeMenu">
```

Comment:
Some of these components have the closeMenu and openMenu methods, but are not watching for the Menu to emit it. So it never calls.